tests/autoyast/installation.pm

```diff
@@ -267,6 +268,12 @@ sub run {
             wait_screen_change { send_key 'alt-u' };
             next;
         }
+        elsif (match_has_tag('salt-formula-motd-setup')) {
```

Comment:
better exclude the needle from the array after matching to avoid unexpected errors.

Reply:
Could you explain what unexpected errors would you expect?

Reply:
needle can be matched several times in the loop, so we `send_key` several times without any control, might not happen anything though, that is the approach that is followed in the file, to exclude the needle after using it.